src/tools/constants.rs

```diff
@@ -1,7 +1,17 @@
 use std::time::Duration;
 
+#[cfg(doc)]
+use crate::setup::testnet::TestNet;
+#[cfg(doc)]
+use crate::tools::inner_node::InnerNode;
+#[cfg(doc)]
+use crate::tools::synth_node::SyntheticNode;
```

Comment:
I'd suggest using these paths in the docs themselves: e.g. `[Testnet](crate::setup::testnet::TestNet)`. `cfg_if` is another option but that adds a dependency, so I'd avoid that for these cases.

Reply:
Done